connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -656,6 +654,16 @@ object SparkSession extends Logging {
       this
     }
 
+    /**
+     * Add an interceptor to be used during channel creation.
+     *
+     * Note that interceptors added last are executed first by gRPC.
```

Comment:
Please check in other parts of spark for proper doc strings. It needs as well a since version marker.

Reply:
done